src/Resources/ResourceManager/Entities/Policy/PolicyExemptionProperties .cs

```diff
@@ -0,0 +1,68 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Entities.Policy
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+    using System;
+
+    /// <summary>
+    /// The policy exemption properties.
+    /// </summary>
+    public class PolicyExemptionProperties
+    {
+        /// <summary>
+        /// Gets or sets the policy assignment Id associate with the policy exemption.
+        /// </summary>
+        [JsonProperty(Required = Required.Always)]
+        public string PolicyAssignmentId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy definition reference Id when the associated policy assignment is an assignment of policy set.
```

Comment:
Id should be plural (Ids), since this is a collection.

---
In reply to: [589725976](https://github.com/Azure/azure-powershell/pull/14474#discussion_r589725976) [](ancestors = 589725976)